pallets/replace/src/benchmarking.rs

```diff
@@ -151,8 +156,15 @@ benchmarks! {
 		VaultRegistry::<T>::issue_tokens(&vault_id, &amount).unwrap();
 		VaultRegistry::<T>::try_increase_to_be_replaced_tokens(&vault_id, &amount).unwrap();
 
-		// TODO: check that an amount was actually withdrawn
+		let vault : DefaultVault::<T> = VaultRegistry::<T>::get_vault_from_id(&vault_id).expect("should return a vault");
```

Comment:
as stated in t[his comment](https://github.com/pendulum-chain/spacewalk/issues/417#issuecomment-1943764696): 
added a `verify` to check that the `to_be_replaced_tokens` was updated.